iot-hub/Samples/device/PnpDeviceSamples/PnpConvention/PnpConvention.cs

```diff
@@ -202,95 +195,145 @@ public static string CreateComponentPropertyPatch(string componentName, IDiction
         }
 
         /// <summary>
-        /// Create a key-embedded value property patch for updating device properties. Embedded value property updates are
-        /// sent from a device in response to a service-initiated read-write property update.
+        /// Creates a response to a write request on a device property.
         /// </summary>
-        /// <remarks>
-        /// A property is either read-only or read-write from a service perspective. All properties are read-write from a
-        /// device's perspective. For a root-level property update, the patch is in the format:
-        /// <code>
-        /// {
-        ///   "samplePropertyName": {
-        ///     "value": 20,
-        ///     "ac": 200,
-        ///     "av": 5,
-        ///     "ad": "The update was successful."
-        ///   }
-        /// }
-        /// </code>
-        ///
-        /// For a component-level property update, the patch is in the format:
-        /// <code>
-        /// {
-        ///   "sampleComponentName": {
-        ///     "__t": "c",
-        ///     "samplePropertyName": {
-        ///       "value": 20,
-        ///       "ac": 200,
-        ///       "av": 5,
-        ///       "ad": "The update was successful."
-        ///     }
-        ///   }
-        /// }
-        /// </code>
-        /// </remarks>
-        /// <param name="propertyName">The property name, as defined in the DTDL interface.</param>
-        /// <param name="unserializedPropertyValue">The unserialized property value, in the format defined in the DTDL interface.</param>
-        /// <param name="ackCode">The acknowledgment code from the device, for the embedded value property update.</param>
-        /// <param name="ackVersion">The version no. of the service-initiated read-write property update.</param>
-        /// <param name="unserializedAckDescription">The serialized description from the device, accompanying the embedded value property update.</param>
-        /// <param name="componentName">The name of the component in which the property is defined. Can be null for property defined under the root interface.</param>
-        /// <returns>The property patch for embedded value property updates for read-write properties.</returns>
-        public static string CreatePropertyEmbeddedValuePatch(
+        /// <param name="propertyName">The name of the property to report.</param>
```

Comment:
nit: the sample patch format would be useful here, and in other overloads below. It will help understand the final structure that we are going for, with each of the dictionaries.

Reply:
yep, i should have included it. thanks